deployment/network-operator/templates/tests/test-rping.yaml

```diff
@@ -1,49 +0,0 @@
-{{- if and .Values.deployCR .Values.rdmaSharedDevicePlugin .deploy .Values.secondaryNetwork.deploy .Values.secondaryNetwork.multus.deploy .Values.secondaryNetwork.cniPlugins.deploy }}
```

Comment:
Were these tests being used anywhere? They won't work after this change, but I'm wondering if this change means some loss in testing coverage.